secure_fw/partitions/firmware_update/bootloader/mcuboot/tfm_mcuboot_fwu.c

```diff
@@ -308,7 +307,7 @@ psa_status_t fwu_bootloader_install_image(bl_image_id_t bootloader_image_id,
     struct image_version image_ver = { 0 };
     const struct flash_area *fap_secondary;
     struct image_header hdr_secondary;
-    uint32_t boot_magic[BOOT_MAGIC_ARR_SZ];
+    uint8_t boot_magic[BOOT_MAGIC_SZ];
```

Comment:
I don't see the reason for this change, can you explain it a bit more clearly. 
Directed towards the whole file and not only this line.